app/src/components/layout.js

```diff
@@ -1,17 +1,9 @@
-/**
- * Layout component that queries for data
- * with Gatsby's useStaticQuery component
- *
- * See: https://www.gatsbyjs.org/docs/use-static-query/
- */
-
 import React from "react"
 import PropTypes from "prop-types"
 import { useStaticQuery, graphql } from "gatsby"
+import Alert from "react-bootstrap/Alert"
```

Comment:
I thought 'Alert' was a tag that generated 'alert' but it was just a box design!

Reply:
Understandable. I think the name `Alert` is somewhat misleading.